mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2318,15 +2321,17 @@ func TestAccContainerCluster_withMonitoringConfig(t *testing.T) {
 				ResourceName:      "google_container_cluster.primary",
 				ImportState:       true,
 				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{"min_master_version"},
 			},
 			// Back to basic settings to test setting Prometheus on its own
 			{
-				Config: testAccContainerCluster_basic(clusterName),
+				Config: testAccContainerCluster_basic_1_23_8(clusterName),
 			},
 			{
 				ResourceName:      "google_container_cluster.primary",
 				ImportState:       true,
 				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{"min_master_version"},
```

Comment:
I'm surprised this needs to be ignored - is the format changed slightly or something?

Reply:
This value is never set in state. 
https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/resource_container_cluster.go#L1914

It's possible it may be able to or there may be challenges there. I think thats beyond the scope or this pr. I don't think this is something we need to address/investigate but if problems do arise we could then take a look.